package.json

```diff
@@ -34,14 +35,10 @@
     "@turf/point-on-feature": "^6.3.0",
     "a-color-picker": "^1.2.1",
     "chroma-js": "^2.1.0",
-    "cookie-parser": "^1.4.5",
-    "dotenv": "^8.2.0",
-    "express": "^4.17.1",
     "flatpickr": "^4.6.3",
     "jwt-decode": "^2.2.0",
     "markdown-it": "^11.0.1",
     "markdown-it-anchor": "^6.0.0",
-    "nodemon": "^2.0.7",
```

Comment:
nodemon is also quite useful for us to run it locally

Reply:
I see, no problem... from the top of my head, I did a search and it wasn't used anywhere, hence I removed it. But no problem leaving it, as it's already under devDependencies.

Reply:
Nice, yeah
it's only used to start the app
It's purpose is to automatically restart the app with new changes... Node v18 is now doing that by default, but I don't think we are using that version of node here

Reply:
No problem, I'll leave it there